challenge.js

```diff
@@ -42,7 +43,7 @@
  */
 
 // ANSWER
-
+element = document.querySelector('E');
```

Comment:
We're just using 'E' to refer to that particular element - it's not an ID, a class, or an element name. One possible selector that would return element E would be `div > p` - it would select all `<p>` elements that are immediate children of `<div>` elements.